iohk-monitoring/src/Cardano/BM/Output/Monitoring.lhs

```diff
@@ -208,4 +214,13 @@ evalMonitoringAction mmap logObj@(LogObject logname _ _) =
             : acc
     -- catch all
     updateEnv env _ = env
+
+    performAction (CreateMessage sev alertMessage) = do
+        lometa <- mkLOMeta sev Public
+        Trace.traceNamedObject sbtrace (lometa, MonitoringEffect (MonitorAlert alertMessage))
```

Comment:
the alert message is static as defined in configuration. could you add a textual representation of the measurable that triggered the monitor and also show the comparison?